YouTube.js

```diff
@@ -166,22 +170,40 @@ var testCases = [
 				"libraryCatalog": "YouTube",
 				"runningTime": "423 seconds",
-				"shortTitle": "HD Charlie Chaplin - The Great Dictator",
-				"url": "https://www.youtube.com/watch?v=9-cM5FLTGhU&feature=youtube_gdata_player",
+				"url": "https://www.youtube.com/watch?v=9-cM5FLTGhU",
 				"attachments": [],
-				"tags": [],
+				"tags": [
+					"1940",
+					"Chaplin",
+					"Charlie Chaplin",
+					"Charlie Chaplin (Author)",
+					"Dictator",
+					"Great",
+					"HD",
+					"High-definition Video (Film Format)",
```

Comment:
though it's interesting that these keywords didn't get split up by word... Maybe the Zotero video is the exception, not the rule.